pyramid/tests/test_scripts/test_pserve.py

```diff
@@ -1,9 +1,11 @@
+import mock
```

Comment:
We don't currently use mock in the the Pyramid tests, instead we create dummy objects that have just the functionality required.

I don't think it is a great idea to add yet another dependency...